props/saber_BC_buttons.h

```diff
@@ -905,30 +923,37 @@ class SaberBCButtons : public PROP_INHERIT_PREFIX PropBase {
       SaberBase::DoBeginLockup();
       return true;
 
-// Battle Mode
-#ifndef BC_NO_BM
+// Spam Blast toggle - pointing up
+// Battle Mode toggle - NOT pointing up
   #if NUM_BUTTONS == 1
     case EVENTID(BUTTON_POWER, EVENT_THIRD_HELD, MODE_ON):
   #else
   // 2 button
     case EVENTID(BUTTON_AUX, EVENT_CLICK_SHORT, MODE_ON | BUTTON_POWER):
   #endif
+      if (fusor.angle1() >  M_PI / 3) {
+        sound_library_.SayZoomingIn();
+#ifndef SPAM_BLAST
```

Comment:
wat?

Reply:
Well, I was trying to avoid redundancy, but couldn't have the case in a regular if statement, so I went with #ifdef #undef method. But I guess that's not proper, so back to repeated code.

Reply:
It might be proper, if I understood why you're doing that.
It just looks weird.

Also, it is in fact possible to have a case statement inside a regular if statement, but it won't make it conditional, it will just jump directly to it.